include/infinispan/hotrod/RemoteCache.h

```diff
@@ -267,7 +268,11 @@ template<class K, class V> class RemoteCache: private RemoteCacheBase
 
     /**
      * Asynchronous version of put()
-     *    See the synchronous doc for parameters not explained here
+     * \param key  key with which the specified value is to be associated
+     * \param val  value value to be associated with the specified key
+     * \param lifespan  lifespan of the entry.  Negative values are interpreted as unlimited lifespan.
+     * \param maxIdle  the maximum amount of time this key is allowed to be idle for before it is considered as
```

Comment:
the maximum amount of time this key is allowed to be idle for before-> the maximum amount of time this key is allowed to be idle before

You can use this for all of the `maxIdle` parameters